### DISCUSSION TITLE
jupyter connecting problem

### DISCUSSION THREAD

**Discussion by Maid-Unknown** [cooperrc/computational-mechanics#145] [Q&A]
I cannot connect with the server since 6 pm. Is it because I open too many windows? I try to refresh the page and restart the computer, but that is no help.  The page said time out.

**Comment by johnf329** [comment]
Several of us have also been experiencing this issue.

**Comment by aliciasle** [comment]
I was having that issue earlier, but it seems to be fine now.
Just make sure to save frequently!

**Comment by dylanbrochard** [comment]
I've been having the same issue, restarting my computer was the only viable solution I found

**Comment by avelineim** [comment]
The server has been on and off for me all day, changing connection sources (between home wifi and mobile hot spots) has helped a little, but has not been a long term solution.

**Comment by Maid-Unknown** [comment]
It is still hard to connect. I cannot work on the project or homework. sad

**Comment by cooperrc** [comment]
sorry about the extra IT headaches!

I'm working with UConn IT to get the server back online, but as a backup I've added everyone to my other Jupyter server:
https://ugmelab.uconn.edu

To get the course materials onto the server:
1. log in to ugmelab
2. click "New" -> Terminal
3. type `cd work`
4. type `git clone https://github.com/cooperrc/computational-mechanics` (or use your own repository url)
5. open the Module_01 folder to find HW_01 and the projects folder to find `01_Getting-started-project`
